code/game/atoms_movable.dm

```diff
@@ -721,6 +721,10 @@
 /atom/movable/proc/can_apply_inertia()
 	return (!src.anchored && !(src.pulledby && src.pulledby.Adjacent(src)))
 
+//Called when somebody begins to pull this atom
+/atom/movable/proc/get_pulled(mob/living/L)
```

Comment:
I do like this idea, though perhaps a more intuitive name could be found? I didn't immediately know what it did when I saw it elsewhere in the changes.